release/steps/am-release-alpha-pipeline.mjs

```diff
@@ -15,7 +15,8 @@ export class ReleaseAlphaVersion extends AbstractStepRelease {
             gio_action: 'release-alpha-version',
             rc_requested: true,
             tag_latest: false,
-            tag_latest_support: false
+            tag_latest_support: false,
+            create_maintenance_version_branch: false
```

Comment:
todo: we need a parameter here to specify if we want to crearte a branch or not. During code freeze, we need to create a branch.